inc/thorvg.h

```diff
@@ -157,7 +157,7 @@ enum class FillRule : uint8_t
 enum class CompositeMethod : uint8_t
 {
     None = 0,           ///< No composition is applied.
-    ClipPath,           ///< The intersection of the source and the target is determined and only the resulting pixels from the source are rendered.
+    ClipPath,           ///< The intersection of the source and the target is determined and only the resulting pixels from the source are rendered. Note that ClipPath only supports the Shape type.
```

Comment:
The term "Shape type" sufficiently conveys that ClipPath works specifically with shapes, and adding "targets" is redundant since the sentence already specifies the interaction between the source and the target.